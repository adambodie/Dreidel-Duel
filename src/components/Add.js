import React, { Component } from "react";
import styles from "./main.module.css";

export default class Add extends Component {
	constructor(props) {
		super(props);
		this.handleChange = this.handleChange.bind(this);
	}

	handleChange(e) {
		this.props.onAddChange(e.target.value);  
	}

	render() {
		return (
			<button className={styles.action} onClick={this.handleChange} disabled={this.props.disabled}>
				Add Coin
			</button>
		);
  }
}
