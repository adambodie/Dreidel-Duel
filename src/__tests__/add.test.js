import React from "react"
import renderer from "react-test-renderer"
import Add from "../components/add"

describe("Add", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<Add />).toJSON()
    expect(tree).toMatchSnapshot()
  }))
