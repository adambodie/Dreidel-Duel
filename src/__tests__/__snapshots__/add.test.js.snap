// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add renders correctly 1`] = `
<button
  className="addAction"
  onClick={[Function]}
>
  Add Coin
</button>
`;
